packages/@okta/migrate-from-jekyll/index.js

```diff
@@ -225,7 +225,7 @@ function run() {
   fs.writeFileSync(docsRoot + '/conductor.yml', 'redirects: \r\n')
   //Build conductor redirects
   redirects.forEach((redirect) => {
-    fs.appendFileSync(docsRoot + '/conductor.yml', ' - from: ' + redirect.path + '\r\n   to: ' + redirect.redirect + '\r\n')
+    fs.appendFileSync(docsRoot + '/conductor.yml', '  - from: ' + redirect.path + '\r\n    to: ' + redirect.redirect + '\r\n')
```

Comment:
When spacing is important I prefer template literals:
```
fs.appendFileSync(`${docsRoot}/conductor.yml`, `  - from: ${redirect.path}\r\n    to: ${redirect.redirect}\r\n`)
```

Also, is the \r\n accurate?  wouldn't that be based on the current file system?

Reply:
This only needs to work for my machine during migration.   This will not run once we migrate.   single use script